fastapi/README.md

```diff
@@ -10,14 +10,22 @@ _Note: The included Dockerfiles are not ready for production. See the appropriat
 
 1. Have a [Userfront](https://userfront.com/) account
 1. Have docker installed
-1. Have the following set as environment variables:
+1. Copy .env.example into a file called .env.local
+1. Have the following set as environment variables in .env.local:
     1. USERFRONT_PUBLIC_KEY (Can be found at https://userfront.com/dashboard/jwt )
-        > export USERFRONT_PUBLIC_KEY="-----BEGIN PUBLIC KEY----- ... -----END PUBLIC KEY-----"
```

Comment:
this feels like a change for personal preference.

Reply:
Since this is started with `docker compose up` wouldn't the original instructions not pass the env variables in?